package.json

```diff
@@ -267,6 +268,7 @@
     "@storybook/react": "^6.3.12",
     "@storybook/storybook-deployer": "^2.8.10",
     "@storybook/theming": "^6.3.0",
+    "@svgr/cli": "^6.3.1",
```

Comment:
SVGR library the allows for the generation of react component from svg files
https://react-svgr.com/docs/getting-started/